op-challenger/game/fault/types/position.go

```diff
@@ -1,15 +1,21 @@
 package types
 
-import "fmt"
+import (
+	"fmt"
+	"math/big"
+)
 
 // Position is a golang wrapper around the dispute game Position type.
 type Position struct {
 	depth        int
-	indexAtDepth int
+	indexAtDepth *big.Int
 }
 
-func NewPosition(depth, indexAtDepth int) Position {
-	return Position{depth, indexAtDepth}
+func NewPosition(depth int, indexAtDepth int) Position {
```

Comment:
Introduced a new temporary `NewPositionFromUint64` method to minimize the diff here since there are quite a lot of places that construct new positions and importing and creating new big ints increases the diff.

In a stacked follow-on pr, will remove it.